parts/k8s/kubernetescustomscript.sh

```diff
@@ -165,6 +164,10 @@ echo `date`,`hostname`, endcustomscript>>/opt/m
 mkdir -p /opt/azure/containers && touch /opt/azure/containers/provision.complete
 ps auxfww > /opt/azure/provision-ps.log &
 
+if ! $FULL_INSTALL_REQUIRED; then
+    cleanUpContainerImages
```

Comment:
moved this down to after container runtime is installed to avoid cleanup colliding with docker uninstall (if new version is needed)